Makefile

```
@@ -48,44 +48,44 @@ geth:
 	@echo "Run \"$(GOBIN)/geth\" to launch geth."
 
 # This rule checks out celo-monorepo under MONOREPO_PATH at commit
-# MONOREPO_COMMIT and compiles the system solidty contracts. A softlink is
-# created at ./monorepo so that code in this repo can always access the system
-# contracts with a consistent path.
-prepare-system-contracts: ./monorepo/packages/protocol/build
-
-# If any of the source files found by the find command are more recent than the
+# MONOREPO_COMMIT and compiles the system solidty contracts. It then copies the
+# compiled contracts from the monorepo to the compiled-system-contracts, so
+# that this repo can always access the contracts at a consistent path.
+prepare-system-contracts: $(MONOREPO_PATH)/packages/protocol/build
+	@rm -rf compiled-system-contracts
+	@cp -ar $(MONOREPO_PATH)/packages/protocol/build/contracts compiled-system-contracts
```

The lowercase `-r` is redundant (and in conflict with) the `-a`
```suggestion
	@cp -a $(MONOREPO_PATH)/packages/protocol/build/contracts compiled-system-contracts
```

---

Well spotted!

Although is the `-r` really in conflict with `-a`? For my version of cp it looks like the `-r` is just redundant, since `-a` is shorthand for `-dR --preserve=all` and `-r` is equivalent to `-R`?

---

My version (BSD cp with mac osx) was claiming conflicts.